src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -234,7 +234,7 @@ private function displayGeneralText(SymfonyStyle $io, string $filter = null)
         }
     }
 
-    private function displayGeneralJson(SymfonyStyle $io, $filter)
+    private function displayGeneralJson(SymfonyStyle $io, string $filter = null)
```

Comment:
I think nicolas wants types in privates to be added in 4.4 for easier maintainance

Reply:
I am working doing that throughout the whole codebase. Thigh I am not sure if I will manage to finish it this weekend.

Reply:
see #32786 (still work in progress right now)

Reply:
will you revert the private methods in this PR then?

Reply:
done